spec/models/task_spec.rb

```diff
@@ -24,4 +24,67 @@
       end
     end
   end
+
+  describe "Methods" do
```

Comment:
We didn't group the methods into a describe block in project_spec.rb. Might be worth doing a small commit to update those for consistency.